rasa_core/training/visualization.py

```diff
@@ -333,7 +333,7 @@ def _create_graph(fontsize: int = 12) -> 'networkx.MultiDiGraph':
 
 def sanitize(s):
     if s:
-        return re.sub(r"[^a-zA-Z0-9\s_-]", "", s)
```

Comment:
but this removes the existing replace of characters that break the dot file format.